docs/projects/fps/tutorial.md

```diff
@@ -404,94 +560,134 @@ namespace Fps
+
+<%(#Expandable title="<b>4. Launch a server-worker and client-worker in-editor.</b>")%>
+
+With the `FPS-Development` Scene open in your Unity Editor, select the Unity `Play` button.
+
+<%(/Expandable)%>
+
+<%(#Expandable title="<b>5. Launch a built-out <code>UnityClient</code> worker-instance</b>.")%>
 
-<%(#Expandable title="4. Launch a client in-editor.")%>With the `FPS-Development` Scene open in your Unity Editor, select the Unity `Play` button.<%(/Expandable)%>
+From the Unity Editor menu, select **SpatialOS** > **Launch standalone client**.
 
-<%(#Expandable title="5. Launch a built-out `UnityClient` worker.")%>From the **SpatialOS** menu, select **Launch standalone client**.
+This will launch an instance of your `UnityClient` in a separate window. This uses the built-out `UnityClient` worker, so make sure you have performed a "Build UnityClient for local" as in step 1.
+
+<%(/Expandable)%>
 
-This will launch an instance of your `UnityClient` in a separate window. This uses the built-out `UnityClient` worker, so make sure you have performed  a "Build UnityClient for local" as in step 1.<%(/Expandable)%>
+<%(#Expandable title="<b>6. Use one client to shoot the other.</b>")%>
 
-<%(#Expandable title="6. Use one client to shoot the other.")%>To see the effects of a health pack restoring a player's health it's a good idea to damage them first. Particularly if you made the optional changes to enforce the maximum health for players, you'll want to confirm that the health pack isn't disappearing without performing its health-giving duty.
+To see the effects of a health pack restoring a player's health you'll need to damage them first.
 
 This may require some switching between the editor and your standalone client, but you should be able to steer one player entity to the other and shoot them a few times.
 
 You can use the SpatialOS inspector to help you find where the two players are, and navigate them to the same location. You can also select the icon for the damaged player and, in the right-side of the SpatialOS inspector, view the component data for the player's `Health` component to confirm that damage from the gunfire has been applied.<%(/Expandable)%>
 
-<%(#Expandable title="7. Use the inspector to check the damage has been applied.")%>When a local deployment is running you can open the SpatialOS local inspector in your browser: http://localhost:21000/inspector/
+<%(#Expandable title="<b>7. Use the inspector to check the damage has been applied.</b>")%>
+
+When a local deployment is running you can open the SpatialOS local inspector in your browser: http://localhost:21000/inspector/
 
 By selecting the visual marker for an entity you can view its component values in the right-side panel, by expanding the `Components` section.
 
 Component values can be found by expanding the namespace for that component. For `Player` health you can find this under the namespace **improbable** > **gdk** > **health** > **HealthComponent** > **health**.
```

Comment:
not to self for future: if we can download and share workspaces with inspector v2 - we could provide one that shows the health information by default instead of having to click through

Reply:
That would be awesome